installation

I am having difficulty installing. I tried all the methods mentioned here. but none of them is working. I am using CentOS. 
I installed dependencies and cuda toolkit using conda .
1. Cloned from github repo but when I tried to execute cmake command, i got an error saying no folder build found. I want to know if I am missing something here. 
2. I tried to use conda to install Uni-dock using "**_conda install -c https://conda.mlops.dp.tech/caic -c conda-forge unidock_**" but again got an error "**_conda.core.prefix_data:_load_single_record(216): Ignoring malformed prefix record at: home/miniconda3/conda-meta/unidock-1.0.0-5_5.json_**"
3. Then i tried downloading "unidock-1.0.0-5_5.tar.bz2" file and run "conda install -c conda-forge unidock-1.0.0-5_5.tar.bz2" command. the command ran without errors but when I try to execute "unidock --help", I get an error unknown command. 

I am new to this. So, it will be great if someone can help. thanks!

Thanks @caic99. 

When I execute the cmake command, this is the error I got.
CMake Error: The source directory "/home/users/dinesh/Uni-Dock/build" does not exist.
Specify --help for usage, or press the help button on the CMake GUI

> Thanks @caic99.
> 
> When I execute the cmake command, this is the error I got. CMake Error: The source directory "/home/users/dinesh/Uni-Dock/build" does not exist. Specify --help for usage, or press the help button on the CMake GUI

@dinu09 Please also provide your input command

@caic99 I entered unidock subdirectory and executed the **_cmake -B build_** command. 

CMake Error: The source directory "/home/users/dinesh/Uni-Dock/unidock/build" does not exist.

--- a/unidock/README.md
+++ b/unidock/README.md
@@ -30,33 +30,32 @@ Uni-Dock supports NVIDIA GPUs on Linux platform.
 
 1. Install dependencies
 
-- Boost >= 1.72
+     - CUDA toolkit: Please refer to the [installation tutorial](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html) provided by nvidia.
+     - CMake >= 3.16
+     - A C++ compiler (should be compatible with NVCC; `g++` works for most cases)
+     - Boost >= 1.72
+          
+          ```bash
+          sudo apt install libboost-system-dev libboost-thread-dev libboost-serialization-dev libboost-filesystem-dev libboost-program-options-dev libboost-timer-dev
+          ```
+          
+          Alternatively, install from [boost source codes](https://www.boost.org/users/download/) or from [anaconda](https://anaconda.org/anaconda/libboost) with `conda install -c anaconda libboost`
 
-```bash
-sudo apt install libboost-system-dev libboost-thread-dev libboost-serialization-dev libboost-filesystem-dev libboost-program-options-dev libboost-timer-dev
-```
+2. Clone repo or retrieve source code from [release page](https://github.com/dptech-corp/Uni-Dock/releases):
 
-Alternatively, install from [boost source codes](https://www.boost.org/users/download/) or from [anaconda](https://anaconda.org/anaconda/libboost) with `conda install -c anaconda libboost`
+   ```bash
+   git clone https://github.com/dptech-corp/Uni-Dock.git
+   ```
 
-- CUDA toolkit: Please refer to the [installation tutorial](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html) provided by nvidia.
-
-2. Clone GitHub repo or retrieve source code from release page
-
-3. Build Uni-Dock using CMake:
-
-```bash
-cmake -B build
-cmake --build build -j4
-```
-
-Or build with Makefile:
-```bash
-cd ./build/linux/release
-make clean
-make -j 4
-```
-
-Add the build directory to your `PATH` environment variable.
+4. Build Uni-Dock using CMake:
+     
+     ```bash
+     cd Uni-Dock/unidock
+     cmake -B build
+     cmake --build build -j4
+     cmake --install build # This will add `unidock` to the default installation path for executables.
+                           # Otherwise, prepend the building directory to your `PATH` environment variable.
+     ```
 
 ### Using binary
 

